### ISSUE TITLE
Something made me drop my weapon

### ISSUE THREAD

**Issue by rpherbig** [rpherbig/dr-scripts#255]
```
[combat-trainer: new skill needed for training 6:34:35]

[combat-trainer: checking weapons as "Bow"!=Offhand Weapon]

[combat-trainer]>wear my longbow

You sling an osage battle longbow over your shoulder.

[combat-trainer]>get my throwing.hammer

You get a steel throwing hammer from inside your backpack.
The rock guardian closes to pole weapon range on you!

[combat-trainer]>swap

The rock guardian closes to pole weapon range on you!

[combat-trainer]>skin

You move a steel throwing hammer to your left hand.
The rubble cannot be skinned.

[combat-trainer: adding rubble to no skin list: []]

[combat-trainer]>drop my hammer

You drop a steel throwing hammer.

[combat-trainer]>loot

You search the pile of rubble.
Lodged into it was a boar-tusk arrow, a boar-tusk arrow, a boar-tusk arrow, a boar-tusk arrow, and a boar-tusk arrow.
The rubble was carrying 2 copper coins, 4 bronze coins, and 2 silver coins!
A pile of rubble crumbles into dark minerals and sifts in between the cracks of the rocks until it's completely gone.
```

**Comment by rpherbig** [comment]
Something else interesting: it did `attack left` for a while since it was trying to train Offhand, but since it had no weapon only Brawling got exp. This showed up a while later once it tried to switch skills:
```

[combat-trainer: checking weapons as "Offhand Weapon"!=Large Edged]

[combat-trainer]>stow my hammer

Stow what?  Type 'STOW HELP' for details.

[hunting-buddy: 15 minutes of hunting remaining]

Your pulse shudders and you begin to lose the violent rage churning inside!
You feel fully rested.

[combat-trainer: *** No match was found after 15 seconds, dumping info]

[combat-trainer: messages seen length: 7]

[combat-trainer: message: Stow what?  Type 'STOW HELP' for details.]

[combat-trainer: message: Your pulse shudders and you begin to lose the violent rage churning inside!]

[combat-trainer: message:  Python (very long time)]

[combat-trainer: message:  Wildfire (somewhat crazed)]

[combat-trainer: message: You feel fully rested.]

[combat-trainer: message:  Python (very long time)]

[combat-trainer: message:  Wildfire (somewhat crazed)]

[combat-trainer: checked against [/You put .*hammer/i, /You .* unload/i]]
```

Apparently `stow my hammer` didn't find the hammer since it was on the ground?

**Comment by rpherbig** [comment]
Later it tried to cycle back to Offhand:
```
[combat-trainer]>get my throwing.hammer

What were you referring to?

[combat-trainer: Could not find throwing.hammer anywhere.]

[combat-trainer]>attack left
```

**Comment by rpherbig** [comment]
I'm not sure if it is related, but hunting-buddy sent the stop signal and combat-trainer never stopped.

**Comment by rpherbig** [comment]
I think this is fixed between bbb2af0848b258803f12ecbeba58a4b878a5fbd6 and 1bceb5ac7f2ed78487242a86425da35daab6abed
